api4/config_test.go

```diff
@@ -471,6 +471,8 @@ func TestUpdateConfigDiffInAuditRecord(t *testing.T) {
 	})
 	require.Equal(t, timeoutVal+1, *cfg.ServiceSettings.ReadTimeout)
 
+	require.NoError(t, logFile.Sync())
```

Comment:
Shouldn't this happen after the flush?

Reply:
I checked that. Flush actually closes the file descriptor. So it should be synced before it.

Reply:
Ah my bad, I was checking the wrong thing. Yes, this should happen after the flush.